pkg/trace/stats/concentrator_test.go

```diff
@@ -48,6 +51,85 @@ func testSpan(spanID uint64, parentID uint64, duration, offset int64, service, r
 	}
 }
 
+// testMeasuredSpan is a function that can make measured spans as test fixtures.
+func testMeasuredSpan(spanID uint64, parentID uint64, duration, offset int64, name, service, resource string, err int32) *pb.Span {
```

Comment:
We could have this function use `testSpan` to save duplication. The only thing is that we'd need to mutate some of the fields already set on the returned span (e.g., `s.Name = name`).